niet/__main__.py

```diff
@@ -48,16 +56,26 @@ def get(data, keywords):
         )
         sys.exit(1)
 
+def print_result(res,out_format):
+    if list == type(res):
```

Comment:
I use this way since isinstance seems to have different comportment regarding the version of python. It could be the issue we are facing but I cannot reproduce...

Reply:
Prefer this kind of syntax for validate instance of:
```python
if isinstance(res, list):
```

Reply:
`isinstance` work fine with python2 and python3

https://docs.python.org/2/library/functions.html#isinstance
https://docs.python.org/fr/3.5/library/functions.html#isinstance